sqlmesh/schedulers/airflow/dag_generator.py

```diff
@@ -118,7 +118,7 @@ def _create_cadence_dag_for_snapshot(
 
         with DAG(
             dag_id=dag_id,
-            schedule_interval=snapshot.node.cron,
+            schedule=snapshot.node.cron,
```

Comment:
It looks like Airflow 2.3.0 and prior still relies on `schedule_interval` (https://github.com/apache/airflow/blob/2.3.0/airflow/models/dag.py#L312). So we can't just switch to `schedule`. We should either suppress the warning, or have conditional logic which picks the attribute name based on the Airflow version.

Reply:
Understood. Is the range of supported airflow explicit?

Reply:
Ideally we should support the entire range of the Airflow 2 major release